### PR TITLE
Added Earth in the sky closes #16

### LINKED ISSUE #16: As a Craft player, I want the Earth to appear in the sky so that it is more similar to Minecraft with how the moon appears in the sky at night.

### PR DESCRIPTION
**Pull request by rhartlage10**
Made separate task to make Earth move around the moon. Will implement this in the next pull request in addition to writing tests, adding doxygen comments, and making the code more readable with better variable names. Just getting the basic functionality down first using a lot of the same code as the sky used.

What the Earth looks like:
Screen Shot 2022-04-10 at 5 57 38 PM
.

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -262,6 +262,13 @@ GLuint gen_sky_buffer() {
     return gen_buffer(sizeof(data), data);
 }
 
+GLuint gen_earth_buffer() {
+    GLfloat *data = malloc_faces(8, 1);
+    // make_earth_cube_faces(data, 1, 1, 1, 1, 1, 1, 0, 10*32, 0, 10);
```
Comment (IndecisiveMagician) [inline src/main.c:267]: Delete comment here

File: src/main.c [hunk 2]
```diff
@@ -1739,6 +1746,20 @@ void render_sky(Attrib *attrib, Player *player, GLuint buffer) {
...
+    State *s = &player->state;
+    float matrix[16];
+    set_matrix_3d(
+        matrix, g->width, g->height,
+        0, 0, 0, s->rx, s->ry, g->fov, g->ortho, g->render_radius);
+    glUseProgram(attrib->program);
+    glUniformMatrix4fv(attrib->matrix, 1, GL_FALSE, matrix);
+    glUniform1i(attrib->sampler, 4);
+    draw_triangles_3d(attrib, buffer, 512);
+    //glUniform1f(attrib->timer, time_of_day());
```
Comment (IndecisiveMagician) [inline src/main.c:1759]: Delete comments here if not in use anymore

**Review by IndecisiveMagician — APPROVED** [review]
Everything looks and works nice, but I left some comments on some things that could be changed to improve the code overall. It's not necessary to change now so I will approve your push but make sure to go back and change these as you do your doxygen task.

**Comment by rhartlage10** [comment]
> 

Good idea. I will try to see if I can implement this.

**Pull request merged**